libs/model-ad/util/src/lib/constants.ts

```diff
@@ -3,3 +3,8 @@ export const HELP_URL =
 
 export const MIN_DESKTOP_WIDTH = 768;
 export const SUPPORT_EMAIL = 'modeladexplorer@sagebionetworks.org';
+export const LOADING_ICON_COLORS = {
+  innerColor: '#00C9BA',
+  centerColor: '#6F51C7',
+  outerColor: '#00737C',
+};
```

Comment:
Model-AD and Agora have different colors for the loading icon.  The constants.ts file will specify the specific colors in Model-AD (and in the future Agora) to be passed to the LoadingIconComponent.